src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -431,7 +437,8 @@ void NotationViewInputController::startDragElements(ElementType elementsType, co
 
 void NotationViewInputController::mouseReleaseEvent(QMouseEvent*)
 {
-    if (!m_interactData.hitElement && !m_isCanvasDragged && !m_view->notationInteraction()->isGripEditStarted()) {
+    if (!m_interactData.hitElement && !m_isCanvasDragged && !m_view->notationInteraction()->isGripEditStarted() && 
+        !m_view->notationInteraction()->isDragStarted()) {
```

Comment:
Can't seem to find any case where adding this causes unexpected behaviour and it's definitely necessary for lasso selection to work.